packages/fx-core/src/plugins/resource/frontend/configs.ts

```diff
@@ -62,14 +74,20 @@ export class FrontendConfig {
       solutionConfigs
     );
 
-    let storageName = ctx.config.getString(FrontendConfigInfo.StorageName);
+    let storageName: string | undefined;
+    if (isArmSupportEnabled()) {
+      storageName = getArmOutput(ctx, ArmOutput.frontendStorageName) as string;
```

Comment:
Seems there are two places store the frontend Storage info, Arm output and config. When we shift to ARM, do we still need to write these info to configs? e.g. StorageName, StorageEndpoint...

Reply:
if we shift to ARM, info about azure resource can be loaded from arm output.
But some info is out of arm scope, we may use config as well.
@blackchoey  may help add more info about this.

Reply:
yes, all Azure related information can be retrieved from ARM output. For other information, config is still needed.

Reply:
Then if we totally shift to ARM, we don't need to write Azure related information to config any more. Just to confirm.